internal/server/environment_vars.go

```diff
@@ -19,6 +20,7 @@ func NewEnvironmentVars() EnvironmentVars {
 		SiriusURL:       getEnv("SIRIUS_URL", "http://host.docker.internal:8080"),
 		SiriusPublicURL: getEnv("SIRIUS_PUBLIC_URL", ""),
 		Prefix:          getEnv("PREFIX", ""),
+		BackendUrl:      getEnv("BACKEND_URL", ""),
```

Comment:
this should be `BackendURL` to match naming conventions

Reply:
done